src/model/mixin/lineStyle.js

```diff
@@ -46,7 +46,7 @@ export default {
         var lineType = this.get('type');
         var dotSize = Math.max(lineWidth, 2);
         var dashSize = lineWidth * 4;
-        return (lineType === 'solid' || lineType == null) ? null
```

Comment:
According to <https://github.com/ecomfe/zrender/blob/master/src/graphic/Path.js#L136>
I think default value should better not be a `[0, 0]`, but still `null` is better.

For this issue, probably we can always set a value to `style.lineDash` in method `getLineStyle` above whether line dash is `null` or not.

Reply:
I understand! I just choose the simplest way, but don't know whether this way is the best!